velox/docs/monitoring/stats.rst

```diff
@@ -39,3 +39,25 @@ Memory Arbitration
      - kByte
      - The reclaimed memory bytes of an operator during the memory arbitration.
        This stats only applies for spillable operators.
+
+HashBuild, HashAggregation
+------------------
+.. list-table::
+   :widths: 40 10 50
+   :header-rows: 1
+
+   * - Stats Name
+     - Type
+     - Description
+   * - hashtable.capacity
+     - kNone
```

Comment:
> kNone

What does that mean? Would readers understand this? Let's find a more intuitive way to describe the type of a stat.

Reply:
kNone, kNanos are RuntimeCounter units `enum class Unit { kNone, kNanos, kBytes };`

Shall we use **Count**  or ** Quantity*?